### PR TITLE
use fs.copyFile for performance

### PR DESCRIPTION
**Pull request by briangough**
### Description

The `fs-extra` module has a `copy` method which uses the native fs.copyFile method for performance. It's a useful module.

#### Screenshots

NA

#### Related Issues / PRs

NA

### Review

Performance improvement.

#### Potential Impact

Compiles

#### Manual Testing Performed

- [X] Unit and acceptance tests 

#### Accessibility

NA

### Deployment

NA

#### Deployment Checklist

NA

#### Metrics and Monitoring

NA

#### Who Needs to Know?

@henryoswald

### CODE REVIEW

File: app/js/UrlCache.js [hunk 1]
```diff
@@ -35,8 +36,12 @@ module.exports = UrlCache = {
         if (error != null) {
           return callback(error)
         }
-        return UrlCache._copyFile(pathToCachedUrl, destPath, function (error) {
+        return fse.copy(pathToCachedUrl, destPath, function (error) {
```
Comment (das7pad) [inline app/js/UrlCache.js:39]: Have you thought about calling [`fs.copyFile`](https://nodejs.org/docs/latest-v12.x/api/fs.html#fs_fs_copyfile_src_dest_flags_callback) directly?

fse is doing two extra stat calls we do not need: https://github.com/jprichardson/node-fs-extra/blob/e6a95058c930953113177c9518f57e83cace3e79/lib/copy/copy.js#L30-L36
graceful-fs has some retry queue, which I do not think we want either: https://github.com/isaacs/node-graceful-fs/blob/89dc1330dcd8fa218c5dff92a97d8792b7da6b12/graceful-fs.js#L182-L188

Comment (briangough) [reply app/js/UrlCache.js:39]: I did try that first but found out that fs.copyFile does not create parent directories, unlike fs-extra.copy, so it was not a drop-in replacement.

Comment (das7pad) [reply app/js/UrlCache.js:39]: The existing code does not handle this either :shrug: 

```
> fs=require('fs');fs.writeFileSync('/tmp/foo.txt','BLOB');fs.createReadStream('/tmp/foo.txt').on('error',console.error).pipe(fs.createWriteStream('/tmp/does/not/exist.png'))
> Uncaught [Error: ENOENT: no such file or directory, open '/tmp/does/not/exist.png'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/tmp/does/not/exist.png',
```
```

Comment (briangough) [reply app/js/UrlCache.js]: Ah, you're right - I've switched it to fs.copyFile.   I got confused about where the error was coming from -- I was experimenting with changing the other use of fse.copy in ResourceWriter to fs.copyFile  at the same time.

**Review by das7pad — APPROVED** [review]

**Pull request merged**